wodles/aws/aws_tools.py

```diff
@@ -313,6 +321,8 @@ def arg_validate_security_lake_auth_params(external_id: Optional[str], name: Opt
     if external_id is None:
         error('Used a subscriber but no --external_id provided.')
         sys.exit(21)
+    if profile is None:
+        info('Used a subscriber but no --aws_profile provided, the "default" value will be used.')
```

Comment:
This should be a debug message.
```suggestion
        debug('Used a subscriber but no --aws_profile provided, the "default" value will be used.', 2)
```

Reply:
Fixed in 05a0300aed2b9ee54f63bcbb9af4f4d34531c1ce.